src/solvers/bitwuzla/bitwuzla_conv.cpp

```diff
@@ -704,23 +669,21 @@ expr2tc bitwuzla_convt::get_array_elem(
   uint64_t index,
   const type2tc &subtype)
 {
-  const bitw_smt_ast *ast = to_solver_smt_ast<bitw_smt_ast>(array);
-
-  size_t size;
-  BitwuzlaTerm **indicies, **values, *default_value;
-  bitwuzla_get_array_value(
-    bitw, ast->a, &indicies, &values, &size, &default_value);
+  const bitw_smt_ast *za = to_solver_smt_ast<bitw_smt_ast>(array);
+  unsigned long array_bound = array->sort->get_domain_width();
```

Comment:
Let's use `size_t` instead of `unsigned long`.

Reply:
Implemented as suggested.